react/src/pages/participants/ParticipantsTable.tsx

```diff
@@ -45,69 +56,77 @@ const rows = [
 
 export default function ParticipantsTable({ display }: ParticipantsTableProps) {
     const classes = useStyles();
+    const [showRunner, setRunner] = React.useState(false);
+    const [activeParticipants, setActiveParticipants] = React.useState<Participant[]>([]);
```

Comment:
Every other file imports `useState` via destructured import instead of `React.useState` so it would be best to stay consistent.